### DISCUSSION TITLE
Release schedule

### DISCUSSION THREAD

**Discussion by cfvescovo** [programble/scraper#112] [Q&A] [answered]
As mentioned in #111, version 0.15.0 has just been published.

Going forward, what could be a good release schedule? 
Would you want to see less frequent releases (with more changes in each release) or more frequent ones? 
Should we create a time-based release plan so that a new release is published every x weeks/months, even if no major changes are implemented? An alternative could be feature-based releases or an hybrid approach (regular patches x.y.z and minor/major releases when needed).

**Comment by adamreichold** [comment, marked as answer]
I would be prefer frequent small feature-based releases.
